tests/unit/test_config.py

```diff
@@ -1,3 +1,52 @@
-# this is a good place to create config files, load them, and test that we get the params we expected
-# also a good place to take some configs and test initializing dispatcher objects with them
-# None of this has been done, but you could do it
+import json
+
+import yaml
+
+import pytest
+
+from dispatcher.config import DispatcherSettings, LazySettings, temporary_settings
+from dispatcher.factories import generate_settings_schema
+
+
+def test_settings_reference_unconfigured():
+    settings = LazySettings()
+    with pytest.raises(Exception) as exc:
+        settings.brokers
+    assert 'Dispatcher not configured' in str(exc)
+
+
+def test_configured_settings():
+    settings = LazySettings()
+    settings._wrapped = DispatcherSettings({'version': 2, 'brokers': {'pg_notify': {'config': {}}}})
+    'pg_notify' in settings.brokers
```

Comment:
I think you forgot the assertion. 
```suggestion
    assert 'pg_notify' in settings.brokers
```